include/ed/io/json_reader.h

```diff
@@ -13,7 +13,7 @@ namespace ed
 namespace io
 {
 
-class JSONReader : public Reader
+class JSONReader : public ed::io::Reader
```

Comment:
does this make any difference given the namespaces here? And if not, why leave the namespaces here?

Reply:
No, it doesn't. But as there is also a Reader class in the rapidjson library(https://github.com/tue-robotics/rapidjson/blob/master/include/rapidjson/reader.h#L2233), it better to make it explicit.